### PR TITLE
wayland: attempt to create a working clipboard

### PR DESCRIPTION
**Pull request by james-lawrence**
while these changes don't fully implement a working copy/paste.
I have confirmed that when Clipboard::get_string is invoked the
contents from the system clipboard are returned.

### CODE REVIEW

**Comment by james-lawrence** [comment]
as an aside the reason I think paste isn't working entirely with this change is because of which key codes are being returned... just havent had a chance to dig into it yet to confirm.

**Review by Maan2003 — APPROVED** [review]
Thanks!

File: druid-shell/src/backend/wayland/clipboard.rs [hunk 1]
```diff
@@ -13,45 +13,285 @@
...
+        }
+
+        return None;
+    }
+}
+
+impl TryFrom<(&wl::Display, &wl::GlobalManager)> for Manager {
+    type Error = waylanderr::Error;
+
+    fn try_from(gm: (&wl::Display, &wl::GlobalManager)) -> Result<Self, Self::Error> {
```
[the remark this answers is not in the document]
Comment (Maan2003) [reply druid-shell/src/backend/wayland/clipboard.rs:207]: It doesn't make sense conceptually, it is not trying to **convert** a `Display` and `GlobalManager`, it is creating a new clipboard manager for a Display using a Global Manager

File: druid-shell/src/backend/shared/xkb/keycodes.rs [hunk 2]
```diff
@@ -8,6 +8,7 @@ use super::xkbcommon_sys::*;
 pub fn map_key(keysym: u32) -> Key {
     use Key::*;
     match keysym {
+        XKB_KEY_v => Key::Character("v".to_string()),
```
Comment (james-lawrence) [inline druid-shell/src/backend/shared/xkb/keycodes.rs:11]: so this fixes paste..... but it implies deeper issues in the keymapping logic for x11/wayland

Comment (Maan2003) [reply druid-shell/src/backend/shared/xkb/keycodes.rs:11]: We can have keep this for now

**Pull request merged**